src/WebJobs.Extensions.OpenAI/Search/SemanticSearchConverter.cs

```diff
@@ -33,15 +34,25 @@ class SemanticSearchConverter :
     public SemanticSearchConverter(
         OpenAISDK.OpenAIClient openAIClient,
         ILoggerFactory loggerFactory,
-        ISearchProvider searchProvider)
+        IEnumerable<ISearchProvider> searchProviders,
+        IOptions<OpenAIConfigOptions> openAiConfigOptions)
     {
         this.openAIClient = openAIClient ?? throw new ArgumentNullException(nameof(openAIClient));
         this.logger = loggerFactory?.CreateLogger<SemanticSearchConverter>() ?? throw new ArgumentNullException(nameof(loggerFactory));
 
-        // This will be null if no search provider extension is configured
-        // TODO: Eventually we need to resolve this by name at execution time by name so that we can support
-        //       multiple search providers.
-        this.searchProvider = searchProvider;
+        openAiConfigOptions.Value.SearchProvider.TryGetValue("type", out object value);
+        this.logger.LogDebug("Type of the searchProvider configured in host file: {type}", value);
```

Comment:
I think this should be `LogInformation` since it's very important for us to know when debugging issues with semantic search.